docs/trust-framework-roles/1.0.md

```diff
@@ -0,0 +1,106 @@
+# Trust Framework Roles
+
+Each Member of the Trust Framework must assign two named individuals in the Directory, each responsible for distinct but complementary legal and technical functions required for participation in the Trust Framework and any associated Schemes.
+
+These roles are:
+
+* **Trust Framework Licence Officer (TFLO)** – responsible for legal authority over licensing and consent
+* **Trust Framework Data Officer (TFDO)** – responsible for technical assurance and compliance
+
+Assignment of these roles is mandatory for all Members. If a named individual leaves the organisation or can no longer fulfil the role, the Member must update the Directory without delay to maintain compliance.
+
+The responsibilities and required capabilities of these roles are defined below.
+
+## Dependencies
+
+A Scheme using this specification must also adopt the following specifications:
+
+* [Generic Sensitivity Classes](../generic-sensitivity-classes/1.0.md) (1.0 or later)
+
+## Trust Framework Licence Officer (TFLO)
+
+The Trust Framework Licence Officer has legal authority to make decisions on data sharing, data use, and consent on behalf of the Member organisation. The TFLO is accountable for ensuring that data published or consumed under the Trust Framework complies with relevant licensing terms and consent conditions.
+
+A TFLO:
+
+* Has authority to sign or approve data licences (e.g. Open Data, Shared Data) on behalf of the Member.
+* Provides legal consent for the use of personal or consumer data where required.
```

Comment:
Maybe "signs off on the use or publication of personal or consumer data where required" - just to remove "consent" which gets tangled up with GDPR meaning